run_tests.sh

```diff
@@ -44,3 +44,12 @@ pytest                              \
     ${TEST_RUNNER_EXTRA_ARGUMENTS}
 rm .coverage
 cd ..
+
+cd signing_service/
+pytest -p no:django                 \
```

Comment:
we will have two separate coverage reports... Is this what we want?

Reply:
Yes, one for `concent_api` and one for `signing_service`. I am not sure if that is what we really want but `.coverage` is created in the directory when tests are run, and I am not sure if they can be run from parent directory correctly. Do you want me to check if they can work this way ?

Reply:
We should have one, but if it's hard to achieve, we could deal with it in a separate issue.